room/room-compiler/src/main/kotlin/androidx/room/processor/ProcessorErrors.kt

```diff
@@ -175,6 +175,10 @@ object ProcessorErrors {
     val CANNOT_FIND_UPDATE_RESULT_ADAPTER = "Not sure how to handle update method's " +
         "return type. Currently the supported return types are void, int or Int."
 
+    val USING_MULTIPLE_CONCURRENCY_PATTERNS = "Dao methods that have a suspend modifier " +
```

Comment:
It would be nice, similar to `transactionMethodAsync` to include the conflicting return type. It is especially useful for KAPT since the stub will have a weird signature (`public Object mySuspendQuery(Continuation<? extends LiveData<MyEntity>> cont)`) and might be hard to parse for the user, but by proving the return type and the 'suspend' part of the message they can link things. We also use the word 'deferred' and 'async' through the documentation and other errors, so would be good to include that too for consistency.

Reply:
Done